File: core-data/src/androidTest/java/home/oleg/placesnearme/coredata/FakesStore.java
package home.oleg.placesnearme.coredata;

import java.util.ArrayList;
import java.util.List;

import home.oleg.placesnearme.coredata.model.DetailedVenueDbEntity;
import home.oleg.placesnearme.coredata.model.PhotoDbEntity;
import home.oleg.placesnearme.coredomain.models.Contact;
import home.oleg.placesnearme.coredomain.models.Location;
import home.oleg.placesnearme.coredomain.models.Photo;

public class FakesStore {

    public static DetailedVenueDbEntity getVenue(String id) {
        DetailedVenueDbEntity detailedVenue = new DetailedVenueDbEntity();
        detailedVenue.setId(id);

        Contact contact = new Contact();
        contact.setFormattedPhone("00000");
        detailedVenue.setContact(contact);

        Location location = new Location();
        location.setLng(45.);
        location.setLng(45.);
        detailedVenue.setLocation(location);

        detailedVenue.setDescription("test");
        detailedVenue.setRating(5);
        detailedVenue.setTitle("test");
        return detailedVenue;
    }

    public static List<PhotoDbEntity> getListPhotos() {
        List<PhotoDbEntity> photoDbEntities = new ArrayList<>();
        PhotoDbEntity photoDbEntity = new PhotoDbEntity();

        Photo photo = new Photo();
        photo.setHeight(200L);
        photo.setWidth(200L);
        photo.setPrefix("test");
        photo.setSuffix("test");
        photoDbEntity.setPhoto(photo);

        photoDbEntities.add(photoDbEntity);
        return photoDbEntities;
    }

}

File: core-data/src/androidTest/java/home/oleg/placesnearme/coredata/FakesStore.kt
package home.oleg.placesnearme.coredata

import java.util.ArrayList

import home.oleg.placesnearme.coredata.model.PlaceEntity
import home.oleg.placesnearme.coredata.model.PhotoEntity
import home.oleg.placesnearme.coredomain.models.Contact
import home.oleg.placesnearme.coredomain.models.UserLocation
import home.oleg.placesnearme.coredomain.models.Photo

object FakesStore {

    fun listPhotos(placeId: String): List<PhotoEntity> {
        val photoDbEntities = ArrayList<PhotoEntity>()

        val photo = Photo()
        photo.height = 200L
        photo.width = 200L
        photo.prefix = "test"
        photo.suffix = "test"

        val photoDbEntity = PhotoEntity(
                placeId = placeId,
                photo = photo)
        photoDbEntities.add(photoDbEntity)
        return photoDbEntities
    }

    fun getVenue(id: String): PlaceEntity {
        return PlaceEntity(
                id = id,
                contact = Contact("00000"),
                location = UserLocation(
                        lat = 45.0,
                        lng = 45.0,
                        address = null
                ),
                description = "test",
                rating = 5f,
                title = "test",
                category = null,
                isFavorite = false,
                isHereNow = false
        )
    }

}
